src/Tizen.NUI/src/public/Common/FrameUpdateCallbackInterface.cs

```diff
@@ -141,40 +137,37 @@ protected bool BakeSize(uint id, Vector3 size)
             {
                 return false;
             }
-            bool ret = Interop.FrameUpdateCallbackInterface.FraemCallbackInterfaceBakeSize(proxyIntPtr, id, Vector3.getCPtr(size));
+            bool ret = Interop.FrameUpdateCallbackInterface.FrameCallbackInterfaceBakeSize(proxyIntPtr, id, Vector3.getCPtr(size));
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
             return ret;
         }
 
-
         /// This will be public opened in next tizen after ACR done. Before ACR, need to be hidden as inhouse API.
         [EditorBrowsable(EditorBrowsableState.Never)]
-        protected bool SetScale(uint id, Vector3 scale)
+        protected bool GetScale(uint id, Vector3 scale)
```

Comment:
is out keyword required for scale?

Reply:
As @hinohie replied, out keyword is not required here because Vector3.getCPtr is used.

Reply:
We use `Vector3.getCPtr` Internally. We don't need to use out keyword here.